### PR TITLE
Bootstrapping Ubuntu minions

### PR DESCRIPTION
**Pull request by mateiw**
How to bootstrap an Ubuntu minion using `mgr-create-bootstrap-repo`.

### CODE REVIEW

File: modules/client-configuration/pages/clients-ubuntu.adoc [hunk 1]
```diff
@@ -46,36 +46,98 @@ Some actions are not yet supported:
...
 
+.Procedure: Setting up the software channels for {ubuntu} Client registration
+
+Ensure you have the corresponding {ubuntu} product enabled and required channels have been fully synchronized:
+
+.Ubuntu 18.04
+* Product: {ubuntu} Client 18.04
+* Mandatory channels: [systemitem]``ubuntu-18.04-pool-amd64``
++
+.Ubuntu 16.04
```
Comment (mcalmer) [inline modules/client-configuration/pages/clients-ubuntu.adoc]: Check: "View File" this headline is not there. Some formating problem somewhere ?

File: modules/client-configuration/pages/clients-ubuntu.adoc [hunk 2]
```diff
@@ -46,36 +46,98 @@ Some actions are not yet supported:
...
+====
+
+Next create custom repositories to mirror the {ubuntu} packages (see <<manage-repositories.adoc>>):
+
+For e.g. create repositories for the `main` and `main-updates` components:
+
+* Repository Label: ubuntu-bionic-main
+* Repository URL: http://ubuntumirror.example.com/ubuntu/dists/bionic/main/binary-amd64/
+* Repository Type: deb
+
```
Comment (mcalmer) [inline modules/client-configuration/pages/clients-ubuntu.adoc]: This separation is not visible when you check "View File".

File: modules/client-configuration/pages/clients-ubuntu.adoc [hunk 3]
```diff
@@ -46,36 +46,98 @@ Some actions are not yet supported:
...
+* Repository Label: ubuntu-bionic-main-updates
+* Repository URL: http://ubuntumirror.example.com/ubuntu/dists/bionic-updates/main/binary-amd64/
+* Repository Type: deb
+
+Create custom channels under the `pool` channel and associate them with the repositories created in the previous step.
+
```
Comment (mcalmer) [inline modules/client-configuration/pages/clients-ubuntu.adoc]: Should we add a section about how to create the "Channel"? Important there is to set the correct architecture which I miss here.

**Comment by Loquacity** [comment]
This needs some fairly serious structure editing. I might wait until you've merged it, though, and then go in.

**Comment by jcayouette** [comment]
@Loquacity I think you can just work directly from the branch or? Then we can merge once you have adjusted it?

**Comment by mcalmer** [comment]
@Loquacity : Maybe it would be enough to have this 1:1 for 3.2 docs as well.

**Comment by Loquacity** [comment]
> @Loquacity I think you can just work directly from the branch or? Then we can merge once you have adjusted it? The possibility exists that we would merge, then forget at some point.  Or am I missing something?

It's on Matei's fork, which complicates things. I'm going to merge now, then do another PR to fix the structure, then copy all that across to 3.2 to address https://bugzilla.suse.com/show_bug.cgi?id=1131991. Once I've got the PRs up I'll put links in the email thread from MC, and you can merge and update on your Wednesday morning.

**Comment by Loquacity** [comment]
Please review:
* Beta: https://github.com/SUSE/doc-susemanager/pull/417
* 3.2: https://github.com/SUSE/doc-susemanager/pull/418

**Pull request merged**